### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by SJTGSHIVAM**
Added typography in this module.
As heading were already needed for different pages so most of the things were already in place mager thing was to add documentation only.

### CODE REVIEW

**Review by varnita619 — APPROVED** [review]
Looks good and clean

**Review by samyakshah3008 — APPROVED** [review]
Good job, Shivam! 
Looks good to merge

**Comment by SJTGSHIVAM** [comment]
Merging as I see two approvals on board.

File: atoms/input/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <title>thisUI</title>
+  </head>
+  <body data-prismjs-copy-timeout="500">
+    <header class="tuid__head">
+      <nav class="tuid__nav--top"></nav>
+    </header>
+
+    <div
+      class="tuid__drawer tui__dis--none invis tui__scrollbar--invisible"
+    ></div>
```
Comment (romabulani) [inline atoms/input/index.html:19]: Just a suggestion, use prettier for good indentation

**Review by romabulani — APPROVED** [review]

File: atoms/input/index.html (hunk 1 above)
Comment (SJTGSHIVAM) [reply atoms/input/index.html:19]: @romabulani Thanks for suggesting I have the extension installed and it automatically breaks this into different lines which I probably need to see, if I can configure it to stop doing this unintentional line break.

**Pull request merged**